packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.9.4+10
```

Comment:
@stuartmorgan not sure if we should keep going with `0.9.4+10` or use `0.9.5`. I only find `x.y.z` versioning in [The doc](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates).

Reply:
Oh I found this: 

> The interpretation of each number is just shifted down one slot: going from 0.1.2 to 0.2.0 indicates a breaking change, going to 0.1.3 indicates a new feature, and going to 0.1.2+1 indicates a change that doesn’t affect the public API. For simplicity’s sake, avoid using + after the version reaches 1.0.0

Looks like we should keep bumping `+x` for bug fixes until we reach `1.0.0`

Reply:
The controlling document here is this one (from PR checklist):
> * [...] with an appropriate new version according to the [pub versioning philosophy](https://dart.dev/tools/pub/versioning),

The second to last paragraph under https://dart.dev/tools/pub/versioning#semantic-versions covers this versioning.

Reply:
wow this version conflict resolution is smart! I like that we document not just what the rules are, but also explain our thought process behind these rules. Very helpful and educative.